HWs/HW2/Fibonacci.c

```diff
@@ -0,0 +1,54 @@
+#include <stdio.h>
+#include <stdbool.h>
+
+#define bound 47
+
+int iteratFibonacci(int number)
+{
+    int firstNumber = 0;
+    int secondNumber = 1;
```

Comment:
```suggestion
    int previousNumber = 0;
    int currentNumber = 1;
```
Так названия отражают "смысл" переменных.